### ISSUE TITLE
3D shape analysis of cylindrical objects with two spheres

### ISSUE THREAD

**Issue by samyuu** [blueskythlikesclouds/MikuMikuLibrary#1]
It has recently come to my notice that not all geometry of this type are correctly detected and disposed of.
I suggest we use _coding_ and _algorithms_ to solve this issue in the future.
Any kind of resulting performance hit is well justified.

**Comment by blueskythlikesclouds** [comment]
Just like the cylindrical objects with two spheres, I now find it necessary to dispose of anthropomorphic animals as well.
The _coding_ and _algorithms_ need to be implemented immediately.

**Comment by rakisaionji** [comment]
The truth that matters is you guys has never been respected the dump provider since the first time you guys distributed the dump to some unqualified kids without his consent. And you guys never want to take any responsibility of that. And moreover, he probably will never give you guys updates and new charts.
If I did not make any drama like doxing threats and shits, those videos would be still on YouTube.
And most of all, you guys even write an email to SEGA and troll them. I have the content of that email and it is just enough to trigger the manhunt already if they took it seriously.
Do you remember what is the purpose why the provider send you guys the dump? The provider was just a naïve Japanese arcade enthusiast that have never know about the foreign community, and he has never known how toxic and degenerate it is. His main purpose is seek help to get the game running offline with full functionality and emulate the ALL.net. But what you guys did is just shittalking and trolling and make shits behind his back.
Let me point out how much contribution you guys did: The most work in arcade shits are belong to Samyuu, he is the sole person that worked hard everyday to get the game running. And for modding is Skyth, he is just using his time to make MikuMukuModel for you guys to make furry and nude shits into the game. Brolijah has some key contributions to both projects and even 60fps patch for RPCS3. And Stewie coochie is just testing and try-and-figure shits like a kid. KujiKita made a good but not practical PD_Tool but his code is quite usable. and Waelwindows is just lurking for a year round after finished his quite packed tool but still unusable. And the rest? Just fuking nudeposting, shittalking and trolling new people. After many years of modding, you guys can't even have a good tutorial or any effort to guide new people.
Not to mention that Japanese really hate modding and converting PjD motions and models into MMD.
And you guys have never been thinking back about what you did to Sam lol. Whatever feature come on TLAC or ELAC, where ever it comes first, you guys just be like: "make it for AFT plz" or "make it for PDA plz"... But I'd rather skip talking about Samyuu for now and get to the main point.
Things that you guys have done behind the provider's back did cause a very bad impression about the foreign community to the Japanese community and give the provider great danger that he may be hunted down already.
In conclusion, it is not so surprise that you guys will never have any more updates coming and AFT may never be properly publicized by any of you since no one of you guys have infrastructure to keep those stuff safely and securely.

**Comment by samyuu** [comment]
That's cool and all but we really need to step it up a notch, Project DIVA *Furry Tone* is advancing at a rapid pace and it might soon already be too late.
I suggest periodically screenshotting the model view, reverse Google Image searching the resulting image and looking out for blacklisted keywords. This way we can both ensure to take away the freedom from the user as well as ruining their privacy - it's a two in one solution!
